src/main/atomdb_broker_client_main.cc

```diff
@@ -0,0 +1,56 @@
+#include <signal.h>
+
+#include <iostream>
+#include <string>
+
+#include "AtomDBSingleton.h"
+#include "ServiceBusSingleton.h"
+#include "Utils.h"
+#include "AtomDBBrokerProxy.h"
+#include "Properties.h"
+
+using namespace std;
+using namespace service_bus;
+using namespace atoms;
+using namespace atomdb;
+using namespace atomdb_broker;
+
+
+int main(int argc, char* argv[]) {
+    if (argc < 6) {
+        cerr << "Usage: " << argv[0]
+             << " <client_ip:client_port> <server_ip:server_port> <start_port:end_port> <action> <tokens+> "
```

Comment:
To be consistent with the processor main, rename `server_` to `peer_`,  which is more correct, and put the parameters in the same order.